source/ssb-chart/templates/access-ingress-internal-authorization-policy.yaml

```diff
@@ -0,0 +1,97 @@
+{{- if and .Values.access (and .Values.access.ingress .Values.access.ingress.internal) }}
+{{- range .Values.access.ingress.internal }}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: "{{ template "app.name" $ }}-ingress-{{.namespace }}-{{ .application }}-auth-policy"
+  labels:
+{{ include "default.labels" $ | indent 4 }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "app.name" $ }}
+  action: ALLOW
+  rules:
+    {{- $fromPrincipal := printf "cluster.local/ns/%s/sa/%s-sa" .namespace .application }}
+  {{- if not .rules }}
+    - from:
+        - source:
+            principals: [ "{{ $fromPrincipal }}" ]
+  {{- else }}
+    {{- range .rules }}
+    - from:
+        - source:
+            principals: [ "{{ $fromPrincipal }}" ]
+            {{- if .jwt }}
```

Comment:
If ".jwt", it probably makes sense to require that "issuer" is set.

Reply:
implemented in [410f70e](https://github.com/statisticsnorway/helm-charts/pull/20/commits/410f70e8d9ea0eaafe47f9dfd2ab1ad372e99a02)